pep-0680.rst

```diff
@@ -282,84 +308,94 @@ a Unicode document in Python is ``str``, not ``bytes``.
+Such a parameter is not necessary for the core use cases outlined in the
+`Motivation`_ section. The absence of this can be pretty easily worked around
+using a wrapper class, transformer function, or a third-party library. Finally,
+support could be added later in a backward-compatible way.
 
 
 Removing support for ``parse_float`` in ``tomllib.load[s]``
 -----------------------------------------------------------
 
 This option is not strictly necessary, since TOML floats are "IEEE 754 binary64
-values", which is ``float`` on most architectures. Using ``decimal.Decimal``
-thus allows users extra precision not promised by the TOML format. However, in
-the author of ``tomli``'s experience, this is useful in scientific and financial
-applications. TOML-facing users may include non-developers who are not aware of
-the limits of double-precision float.
+values", which is equivalent to a Python ``float`` on most architectures.
+However, TOML-facing users may include non-developers who are not aware of
+the limits of double-precision float, or need greater precision.
+Using ``decimal.Decimal`` allows users extra precision not promised by the
+TOML format, which in the author of ``tomli``'s experience is particularly
+useful in scientific and financial applications.
 
-There are also niche architectures where the Python ``float`` is not a IEEE-754
-binary64. The ``parse_float`` argument allows users to achieve correct TOML
-semantics even on such architectures.
+There are also niche architectures where the Python ``float`` is not a IEEE 754
+binary64 value. The ``parse_float`` argument allows users to achieve correct
+TOML semantics even on such architectures.
 
 
-Alternative names for module
-----------------------------
+Alternative names for the module
+--------------------------------
 
 Ideally, we would be able to use the ``toml`` module name.
 
 However, the ``toml`` package on PyPI is widely used, so there are backward
 compatibility concerns. Since the standard library takes precedence over third
-party packages, users who have pinned versions of ``toml`` would be broken when
-upgrading Python versions by any API incompatibilities.
-
-To further clarify, the user pins are the specific concern here. Even if we were
-able to get control over the ``toml`` PyPI package and repurpose it as a
-standard library backport, we would still break users who have pinned to
-versions of the current ``toml`` package. This is unfortunate, since pinning
+party packages, libraries and applications who current depend on the ``toml``
+package would likely break when upgrading Python versions due to the many
+`API incompatibilities <Appendix A_>`_, even if they pin their dependency
+versions.
+
+To further clarify, applications with pinned dependencies are of greatest
+concern here. Even if we were able to obtain control of the ``toml`` PyPI
+package and repurpose it for backport of the standard library module,
```

Comment:
Good catch. I went with "for a", since it "PyPI package name" (as I corrected my previous text to read) is used _for_ the backport; it isn't _itself_ the backport.